swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0, age=0):
+        super().__init__(category, condition, age)
+
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
Setting a default value will still allow us to reassign the value. In this case, every `Clothing` instance can change its category to 'Electronics':
```python
pants = Clothing('Electronics', 4.0) 
print(pants.category) # returns 'Electronics'
#what are electronic pants? lol
``` 

We can move the default assignment of `category` to the parent `super()` constructor. This will ensure that every `Clothing` instance will be assigned 'Clothing' as its category.

```python
pants = Clothing(4.0) 
print(pants.category) # returns clothing
```